definitions/procedures/backup/config_files.rb

```diff
@@ -35,7 +35,8 @@ def config_files
       exclude_configs = []
       ForemanMaintain.available_features.each do |feature|
         # exclude proxy as it has special handling later
-        next if feature == feature(:foreman_proxy)
+        next if feature == feature(:instance).proxy_feature
```

Comment:
My guess is `feature(:instance).proxy_feature` will always be equal to `feature(:foreman_proxy)` but we need to exclude also `feature(:capsule)` as we want to avoid collecting its configs at this moment. So the code should be:
```ruby
next if feature == feature(:foreman_proxy)
next if feature(:capsule) && feature == feature(:capsule)
```

Reply:
Sorry, my mistake! I will correct that.